okapi/lib/ClsTbsZip.php

```diff
@@ -15,13 +15,12 @@
 Visit http://www.tinybutstrong.com
 */
 
-
 class ClsTbsZip
 {
     const TBSZIP_DOWNLOAD = 1;   // download (default)
     const TBSZIP_NOHEADER = 4;   // option to use with DOWNLOAD: no header is sent
-    const TBSZIP_FILE     = 8;   // output to file  , or add from file
-    const TBSZIP_STRING   = 32;  // output to string, or add from string
+    const TBSZIP_FILE = 8;   // output to file  , or add from file
+    const TBSZIP_STRING = 32;  // output to string, or add from string
 
```

Comment:
In this small example I think it doesn't matter, but in larger code sections a tabular formatting can improve readability.

Reply:
See for example [these new tables](https://github.com/opencaching/okapi/blob/master/okapi/services/caches/formatters/gpx/WebService.php#L27).

Reply:
> a tabular formatting can improve readability.

but a small change can be exploded to many lines if we add as an example const TBSZIP_STRING_SUPER_LONG = 1024; every other line has to be updated too.

Reply:
I don't see it that strict. In many cases tabular formatting is nice, in some cases it won't work at all, and in other they may be a few strings that don't fit but the rest of the table is aligned. It's an _option_.